packages/frontapp/src/app/components/DonateViaISBN/index.tsx

```diff
@@ -139,13 +140,13 @@ const DonateViaISBN: FC<IPropsViaISBN> = ({
   const handleSubmit = async (e: React.SyntheticEvent) => {
     e.preventDefault();
 
-    dispatch(setIdentifier(valueOfISBN));
+    dispatch(setIdentifier(transformISBNcode(valueOfISBN)));
     if (isMounted) {
       try {
         setIsLoading(true);
         const metaDataOfMaterial = await client.query({
           query: GetMaterialFromMetadataDocument,
-          variables: { identifier: valueOfISBN },
+          variables: { identifier: transformISBNcode(valueOfISBN) },
```

Comment:
I don't recommend do it on the client side. Let's have normalization on the backend side, because there are might be other API calls from the webapp (or other app client). For instance, oz.by, ignores dashes on backend side too.